Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -1141,52 +1176,54 @@ variable {m : Type w} [DecidableEq m] [Fintype m]
 open Matrix
 
 @[simp]
-theorem toMatrix'_comp (Q : QuadraticForm R (m → R)) (f : (n → R) →ₗ[R] m → R) :
+theorem toMatrix'_comp (Q : QuadraticMap R (m → R) R) (f : (n → R) →ₗ[R] m → R) :
     (Q.comp f).toMatrix' = (LinearMap.toMatrix' f)ᵀ * Q.toMatrix' * (LinearMap.toMatrix' f) := by
   ext
-  simp only [QuadraticForm.associated_comp, LinearMap.toMatrix₂'_compl₁₂, toMatrix']
+  simp only [QuadraticMap.associated_comp, LinearMap.toMatrix₂'_compl₁₂, toMatrix']
 
-#align quadratic_form.to_matrix'_comp QuadraticForm.toMatrix'_comp
+#align quadratic_form.to_matrix'_comp QuadraticMap.toMatrix'_comp
 
 section Discriminant
 
-variable {Q : QuadraticForm R (n → R)}
+variable {Q : QuadraticMap R (n → R) R}
 
 /-- The discriminant of a quadratic form generalizes the discriminant of a quadratic polynomial. -/
-def discr (Q : QuadraticForm R (n → R)) : R :=
+def discr (Q : QuadraticMap R (n → R) R) : R :=
   Q.toMatrix'.det
-#align quadratic_form.discr QuadraticForm.discr
+#align quadratic_form.discr QuadraticMap.discr
 
 theorem discr_smul (a : R) : (a • Q).discr = a ^ Fintype.card n * Q.discr := by
   simp only [discr, toMatrix'_smul, Matrix.det_smul]
-#align quadratic_form.discr_smul QuadraticForm.discr_smul
+#align quadratic_form.discr_smul QuadraticMap.discr_smul
 
 theorem discr_comp (f : (n → R) →ₗ[R] n → R) :
     (Q.comp f).discr = f.toMatrix'.det * f.toMatrix'.det * Q.discr := by
-  simp only [Matrix.det_transpose, mul_left_comm, QuadraticForm.toMatrix'_comp, mul_comm,
+  simp only [Matrix.det_transpose, mul_left_comm, QuadraticMap.toMatrix'_comp, mul_comm,
     Matrix.det_mul, discr]
-#align quadratic_form.discr_comp QuadraticForm.discr_comp
+#align quadratic_form.discr_comp QuadraticMap.discr_comp
 
 end Discriminant
```

Comment:
Once again:
- This section should be in the namespace QuadraticForm instead of QuadraticMap.
- Every occurrence of QuadraticMap in this section should be replaced with QuadraticForm.